vcs/vcs.proto

```diff
@@ -53,6 +53,10 @@ message BranchesOptions {
 	// information against the specified base branch. If left blank, then branches will
 	// not include that information and their Counts will be nil.
 	string behind_ahead_branch = 1 [(gogoproto.moretags) = "url:\",omitempty\""];
+
+	// ContainsCommit specifies a commit hash. If non-empty, branches
```

Comment:
Elsewhere we call it a "commit ID", so call it that to be consistent.

In "If non-empty, **branches** will return", you are referring to the `Branches` method, right? Capitalize it.

And I would prefer wording like:

> ContainsCommit filters the list of branches to only those that contain a specific commit ID (if set).

It eliminates ambiguity about whether it filters or adds to the results ("will return" is unclear), and it gets to the point quicker.